tests/common.go

```diff
@@ -129,17 +129,24 @@ func InitInstance() {
 
 // ValidateCleanup checks that there are no resource leaks after the test run
 func ValidateCleanup() {
-	Step(fmt.Sprintf("validate cleanup of resources used by the test suite"), func() {
-		t := func() (interface{}, bool, error) {
-			if err := Inst().V.ValidateVolumeCleanup(); err != nil {
-				return "", true, err
-			}
+	context("For validate clean up", func() {
```

Comment:
nit: change `For validate clean up` to `To validate clean up` so that it reads better.

That is, `To validate clean up, validate cleanup of resources used by the test suite`